roachtest: rust-postgres failed

roachtest.rust-postgres [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/12059860?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/12059860?buildTab=artifacts#/rust-postgres) on master @ [6b08842e45668287861af596c28dce58c352d77e](https://github.com/cockroachdb/cockroach/commits/6b08842e45668287861af596c28dce58c352d77e):


```
(orm_helpers.go:212).summarizeFailed: 
Tests run on Cockroach v23.2.0-alpha.2-dev-6b08842e45668287861af596c28dce58c352d77e
Tests run against rust-postgres 
143 Total Tests Run
48 tests passed
95 tests failed
0 tests skipped
1 test ignored
0 tests passed unexpectedly
28 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: types.inet (unexpected)
--- FAIL: types.test_i8_params (unexpected)
--- FAIL: runtime.cancel_query (unexpected)
--- FAIL: types.test_varchar_params (unexpected)
--- FAIL: test.cancel_query (unexpected)
--- FAIL: types.test_bytea_params (unexpected)
--- FAIL: test.transaction_rollback (unexpected)
--- FAIL: types.test_i64_params (unexpected)
--- FAIL: types.test_f32_params (unexpected)
--- FAIL: test.explicit_close (unexpected)
--- FAIL: types.test_lsn_params (unexpected)
--- FAIL: test.query_prepared (unexpected)
--- FAIL: types.test_slice_wrong_type (unexpected)
--- FAIL: types.test_oid_params (unexpected)
--- FAIL: types.test_text_params (unexpected)
--- FAIL: types.test_f32_nan_param (unexpected)
--- FAIL: test.query_unprepared (unexpected)
--- FAIL: test.transaction_drop (unexpected)
--- FAIL: types.test_bool_params (unexpected)
--- FAIL: types.test_bpchar_params (unexpected)
--- FAIL: test.copy_out (unexpected)
--- FAIL: types.test_f64_nan_param (unexpected)
--- FAIL: types.test_name_params (unexpected)
--- FAIL: types.test_f64_params (unexpected)
--- FAIL: test.transaction_drop_immediate_rollback (unexpected)
--- FAIL: types.test_borrowed_text (unexpected)
--- FAIL: types.test_borrowed_bytea (unexpected)
--- FAIL: types.system_time (unexpected)
For a full summary look at the rust-postgres artifacts 
An updated blocklist (rustPostgresBlockList) is available in the artifacts' rust-postgres log
test artifacts and logs in: /artifacts/rust-postgres/run_1
```
<p>Parameters: <code>ROACHTEST_arch=amd64</code>
, <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=16</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-12059860/rust-postgres/1696499062805/1696499280892)

</p>
</details>
/cc @cockroachdb/sql-foundations
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*rust-postgres.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>


Jira issue: CRDB-32081

roachtest.rust-postgres [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/12074329?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/12074329?buildTab=artifacts#/rust-postgres) on master @ [d926f1e51e23d477bcf3027adc6beac11513e58a](https://github.com/cockroachdb/cockroach/commits/d926f1e51e23d477bcf3027adc6beac11513e58a):


```
(orm_helpers.go:212).summarizeFailed: 
Tests run on Cockroach v23.2.0-alpha.2-dev-d926f1e51e23d477bcf3027adc6beac11513e58a
Tests run against rust-postgres 
143 Total Tests Run
48 tests passed
95 tests failed
0 tests skipped
1 test ignored
0 tests passed unexpectedly
28 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: types.test_borrowed_text (unexpected)
--- FAIL: types.test_f32_nan_param (unexpected)
--- FAIL: test.copy_out (unexpected)
--- FAIL: types.test_bytea_params (unexpected)
--- FAIL: test.transaction_drop_immediate_rollback (unexpected)
--- FAIL: types.test_oid_params (unexpected)
--- FAIL: types.test_bool_params (unexpected)
--- FAIL: test.query_prepared (unexpected)
--- FAIL: types.test_f64_params (unexpected)
--- FAIL: test.transaction_drop (unexpected)
--- FAIL: types.test_f64_nan_param (unexpected)
--- FAIL: types.test_text_params (unexpected)
--- FAIL: types.system_time (unexpected)
--- FAIL: types.test_i8_params (unexpected)
--- FAIL: types.test_i64_params (unexpected)
--- FAIL: types.inet (unexpected)
--- FAIL: types.test_f32_params (unexpected)
--- FAIL: types.test_slice_wrong_type (unexpected)
--- FAIL: test.transaction_rollback (unexpected)
--- FAIL: types.test_lsn_params (unexpected)
--- FAIL: test.cancel_query (unexpected)
--- FAIL: runtime.cancel_query (unexpected)
--- FAIL: types.test_borrowed_bytea (unexpected)
--- FAIL: test.query_unprepared (unexpected)
--- FAIL: test.explicit_close (unexpected)
--- FAIL: types.test_varchar_params (unexpected)
--- FAIL: types.test_bpchar_params (unexpected)
--- FAIL: types.test_name_params (unexpected)
For a full summary look at the rust-postgres artifacts 
An updated blocklist (rustPostgresBlockList) is available in the artifacts' rust-postgres log
test artifacts and logs in: /artifacts/rust-postgres/run_1
```
<p>Parameters: <code>ROACHTEST_arch=amd64</code>
, <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=16</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-12074329/rust-postgres/1696586586089/1696586804391)

</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*rust-postgres.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.rust-postgres [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Coverage_RoachtestNightlyGce/12080468?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Coverage_RoachtestNightlyGce/12080468?buildTab=artifacts#/rust-postgres) on master @ [3b438b4a59ad4759e5bf22ff0b6dd6e678c2be0d](https://github.com/cockroachdb/cockroach/commits/3b438b4a59ad4759e5bf22ff0b6dd6e678c2be0d):


```
(orm_helpers.go:212).summarizeFailed: 
Tests run on Cockroach v23.2.0-alpha.2-dev-3b438b4a59ad4759e5bf22ff0b6dd6e678c2be0d
Tests run against rust-postgres 
143 Total Tests Run
48 tests passed
95 tests failed
0 tests skipped
1 test ignored
0 tests passed unexpectedly
28 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: types.test_lsn_params (unexpected)
--- FAIL: types.test_f64_params (unexpected)
--- FAIL: types.test_i64_params (unexpected)
--- FAIL: test.transaction_drop_immediate_rollback (unexpected)
--- FAIL: test.explicit_close (unexpected)
--- FAIL: types.test_slice_wrong_type (unexpected)
--- FAIL: test.query_unprepared (unexpected)
--- FAIL: types.test_f32_params (unexpected)
--- FAIL: types.test_oid_params (unexpected)
--- FAIL: runtime.cancel_query (unexpected)
--- FAIL: test.query_prepared (unexpected)
--- FAIL: test.cancel_query (unexpected)
--- FAIL: types.system_time (unexpected)
--- FAIL: types.test_f32_nan_param (unexpected)
--- FAIL: types.test_text_params (unexpected)
--- FAIL: types.test_name_params (unexpected)
--- FAIL: test.copy_out (unexpected)
--- FAIL: test.transaction_rollback (unexpected)
--- FAIL: types.test_i8_params (unexpected)
--- FAIL: types.test_bytea_params (unexpected)
--- FAIL: test.transaction_drop (unexpected)
--- FAIL: types.test_bpchar_params (unexpected)
--- FAIL: types.test_varchar_params (unexpected)
--- FAIL: types.test_bool_params (unexpected)
--- FAIL: types.test_f64_nan_param (unexpected)
--- FAIL: types.test_borrowed_bytea (unexpected)
--- FAIL: types.inet (unexpected)
--- FAIL: types.test_borrowed_text (unexpected)
For a full summary look at the rust-postgres artifacts 
An updated blocklist (rustPostgresBlockList) is available in the artifacts' rust-postgres log
test artifacts and logs in: /artifacts/rust-postgres/run_1
```
<p>Parameters: <code>ROACHTEST_arch=amd64</code>
, <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=16</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-12080468/rust-postgres/1696624952788/1696625170091)

</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*rust-postgres.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.rust-postgres [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/12090663?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/12090663?buildTab=artifacts#/rust-postgres) on master @ [981b050f28f406b495b530ec5725fa2c91cd20c5](https://github.com/cockroachdb/cockroach/commits/981b050f28f406b495b530ec5725fa2c91cd20c5):


```
(orm_helpers.go:212).summarizeFailed: 
Tests run on Cockroach v23.2.0-alpha.2-dev-981b050f28f406b495b530ec5725fa2c91cd20c5
Tests run against rust-postgres 
143 Total Tests Run
48 tests passed
95 tests failed
0 tests skipped
1 test ignored
0 tests passed unexpectedly
28 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: test.transaction_rollback (unexpected)
--- FAIL: types.test_i64_params (unexpected)
--- FAIL: types.test_borrowed_bytea (unexpected)
--- FAIL: types.test_f64_nan_param (unexpected)
--- FAIL: types.test_bpchar_params (unexpected)
--- FAIL: types.inet (unexpected)
--- FAIL: types.test_f32_nan_param (unexpected)
--- FAIL: types.test_name_params (unexpected)
--- FAIL: test.cancel_query (unexpected)
--- FAIL: test.query_unprepared (unexpected)
--- FAIL: types.test_oid_params (unexpected)
--- FAIL: types.system_time (unexpected)
--- FAIL: test.copy_out (unexpected)
--- FAIL: test.transaction_drop_immediate_rollback (unexpected)
--- FAIL: types.test_f64_params (unexpected)
--- FAIL: types.test_text_params (unexpected)
--- FAIL: types.test_bool_params (unexpected)
--- FAIL: types.test_lsn_params (unexpected)
--- FAIL: test.explicit_close (unexpected)
--- FAIL: types.test_borrowed_text (unexpected)
--- FAIL: types.test_slice_wrong_type (unexpected)
--- FAIL: runtime.cancel_query (unexpected)
--- FAIL: types.test_bytea_params (unexpected)
--- FAIL: test.query_prepared (unexpected)
--- FAIL: test.transaction_drop (unexpected)
--- FAIL: types.test_varchar_params (unexpected)
--- FAIL: types.test_f32_params (unexpected)
--- FAIL: types.test_i8_params (unexpected)
For a full summary look at the rust-postgres artifacts 
An updated blocklist (rustPostgresBlockList) is available in the artifacts' rust-postgres log
test artifacts and logs in: /artifacts/rust-postgres/run_1
```
<p>Parameters: <code>ROACHTEST_arch=amd64</code>
, <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=16</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-12090663/rust-postgres/1696672363054/1696672581988)

</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*rust-postgres.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.rust-postgres [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Coverage_RoachtestNightlyGce/12097524?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Coverage_RoachtestNightlyGce/12097524?buildTab=artifacts#/rust-postgres) on master @ [3b438b4a59ad4759e5bf22ff0b6dd6e678c2be0d](https://github.com/cockroachdb/cockroach/commits/3b438b4a59ad4759e5bf22ff0b6dd6e678c2be0d):


```
(orm_helpers.go:212).summarizeFailed: 
Tests run on Cockroach v23.2.0-alpha.2-dev-3b438b4a59ad4759e5bf22ff0b6dd6e678c2be0d
Tests run against rust-postgres 
143 Total Tests Run
48 tests passed
95 tests failed
0 tests skipped
1 test ignored
0 tests passed unexpectedly
28 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: types.test_i8_params (unexpected)
--- FAIL: types.test_borrowed_bytea (unexpected)
--- FAIL: types.test_oid_params (unexpected)
--- FAIL: test.transaction_drop (unexpected)
--- FAIL: test.explicit_close (unexpected)
--- FAIL: runtime.cancel_query (unexpected)
--- FAIL: types.test_bool_params (unexpected)
--- FAIL: test.transaction_rollback (unexpected)
--- FAIL: types.test_name_params (unexpected)
--- FAIL: types.test_text_params (unexpected)
--- FAIL: types.test_varchar_params (unexpected)
--- FAIL: types.test_f32_nan_param (unexpected)
--- FAIL: types.test_slice_wrong_type (unexpected)
--- FAIL: types.inet (unexpected)
--- FAIL: types.test_bytea_params (unexpected)
--- FAIL: types.system_time (unexpected)
--- FAIL: types.test_bpchar_params (unexpected)
--- FAIL: test.cancel_query (unexpected)
--- FAIL: test.query_prepared (unexpected)
--- FAIL: test.query_unprepared (unexpected)
--- FAIL: test.copy_out (unexpected)
--- FAIL: types.test_f64_nan_param (unexpected)
--- FAIL: types.test_f32_params (unexpected)
--- FAIL: types.test_i64_params (unexpected)
--- FAIL: test.transaction_drop_immediate_rollback (unexpected)
--- FAIL: types.test_f64_params (unexpected)
--- FAIL: types.test_borrowed_text (unexpected)
--- FAIL: types.test_lsn_params (unexpected)
For a full summary look at the rust-postgres artifacts 
An updated blocklist (rustPostgresBlockList) is available in the artifacts' rust-postgres log
test artifacts and logs in: /artifacts/rust-postgres/run_1
```
<p>Parameters: <code>ROACHTEST_arch=amd64</code>
, <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=16</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-12097524/rust-postgres/1696727784598/1696727999148)

</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*rust-postgres.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.rust-postgres [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/12098424?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/12098424?buildTab=artifacts#/rust-postgres) on master @ [76a8d1077d3d29bcf020a48972efd1f77aa892c2](https://github.com/cockroachdb/cockroach/commits/76a8d1077d3d29bcf020a48972efd1f77aa892c2):


```
(orm_helpers.go:212).summarizeFailed: 
Tests run on Cockroach v23.2.0-alpha.2-dev-76a8d1077d3d29bcf020a48972efd1f77aa892c2
Tests run against rust-postgres 
143 Total Tests Run
48 tests passed
95 tests failed
0 tests skipped
1 test ignored
0 tests passed unexpectedly
28 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: types.test_f32_params (unexpected)
--- FAIL: runtime.cancel_query (unexpected)
--- FAIL: types.test_name_params (unexpected)
--- FAIL: types.test_bytea_params (unexpected)
--- FAIL: types.test_slice_wrong_type (unexpected)
--- FAIL: types.test_lsn_params (unexpected)
--- FAIL: test.query_unprepared (unexpected)
--- FAIL: test.cancel_query (unexpected)
--- FAIL: types.test_borrowed_bytea (unexpected)
--- FAIL: types.test_i8_params (unexpected)
--- FAIL: test.explicit_close (unexpected)
--- FAIL: test.query_prepared (unexpected)
--- FAIL: types.inet (unexpected)
--- FAIL: types.test_bpchar_params (unexpected)
--- FAIL: types.test_varchar_params (unexpected)
--- FAIL: test.copy_out (unexpected)
--- FAIL: test.transaction_rollback (unexpected)
--- FAIL: types.system_time (unexpected)
--- FAIL: types.test_f32_nan_param (unexpected)
--- FAIL: types.test_f64_params (unexpected)
--- FAIL: types.test_oid_params (unexpected)
--- FAIL: types.test_f64_nan_param (unexpected)
--- FAIL: types.test_i64_params (unexpected)
--- FAIL: types.test_text_params (unexpected)
--- FAIL: test.transaction_drop (unexpected)
--- FAIL: types.test_borrowed_text (unexpected)
--- FAIL: test.transaction_drop_immediate_rollback (unexpected)
--- FAIL: types.test_bool_params (unexpected)
For a full summary look at the rust-postgres artifacts 
An updated blocklist (rustPostgresBlockList) is available in the artifacts' rust-postgres log
test artifacts and logs in: /artifacts/rust-postgres/run_1
```
<p>Parameters: <code>ROACHTEST_arch=amd64</code>
, <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=16</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-12098424/rust-postgres/1696758280747/1696758509971)

</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*rust-postgres.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.rust-postgres [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/12106584?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/12106584?buildTab=artifacts#/rust-postgres) on master @ [330500d0433bae42bc7c0d6842a427aab3b27f03](https://github.com/cockroachdb/cockroach/commits/330500d0433bae42bc7c0d6842a427aab3b27f03):


```
(orm_helpers.go:212).summarizeFailed: 
Tests run on Cockroach v23.2.0-alpha.2-dev-330500d0433bae42bc7c0d6842a427aab3b27f03
Tests run against rust-postgres 
143 Total Tests Run
48 tests passed
95 tests failed
0 tests skipped
1 test ignored
0 tests passed unexpectedly
28 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: test.query_unprepared (unexpected)
--- FAIL: types.test_oid_params (unexpected)
--- FAIL: types.test_bytea_params (unexpected)
--- FAIL: types.test_name_params (unexpected)
--- FAIL: types.test_borrowed_text (unexpected)
--- FAIL: types.test_f64_params (unexpected)
--- FAIL: types.inet (unexpected)
--- FAIL: test.query_prepared (unexpected)
--- FAIL: test.transaction_rollback (unexpected)
--- FAIL: types.test_f32_nan_param (unexpected)
--- FAIL: test.copy_out (unexpected)
--- FAIL: types.test_i8_params (unexpected)
--- FAIL: types.test_f64_nan_param (unexpected)
--- FAIL: test.explicit_close (unexpected)
--- FAIL: types.test_text_params (unexpected)
--- FAIL: types.test_f32_params (unexpected)
--- FAIL: test.cancel_query (unexpected)
--- FAIL: types.test_varchar_params (unexpected)
--- FAIL: types.test_slice_wrong_type (unexpected)
--- FAIL: types.test_lsn_params (unexpected)
--- FAIL: test.transaction_drop (unexpected)
--- FAIL: runtime.cancel_query (unexpected)
--- FAIL: types.test_borrowed_bytea (unexpected)
--- FAIL: test.transaction_drop_immediate_rollback (unexpected)
--- FAIL: types.system_time (unexpected)
--- FAIL: types.test_i64_params (unexpected)
--- FAIL: types.test_bpchar_params (unexpected)
--- FAIL: types.test_bool_params (unexpected)
For a full summary look at the rust-postgres artifacts 
An updated blocklist (rustPostgresBlockList) is available in the artifacts' rust-postgres log
test artifacts and logs in: /artifacts/rust-postgres/run_1
```
<p>Parameters: <code>ROACHTEST_arch=amd64</code>
, <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=16</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-12106584/rust-postgres/1696844548378/1696844770511)

</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*rust-postgres.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.rust-postgres [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/12121579?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/12121579?buildTab=artifacts#/rust-postgres) on master @ [7e1f909e9f2ddcef2e418d6c2df11f3139a2b85a](https://github.com/cockroachdb/cockroach/commits/7e1f909e9f2ddcef2e418d6c2df11f3139a2b85a):


```
(orm_helpers.go:212).summarizeFailed: 
Tests run on Cockroach v23.2.0-alpha.3-dev-7e1f909e9f2ddcef2e418d6c2df11f3139a2b85a
Tests run against rust-postgres 
143 Total Tests Run
48 tests passed
95 tests failed
0 tests skipped
1 test ignored
0 tests passed unexpectedly
28 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: test.cancel_query (unexpected)
--- FAIL: runtime.cancel_query (unexpected)
--- FAIL: types.inet (unexpected)
--- FAIL: types.test_bool_params (unexpected)
--- FAIL: types.test_bpchar_params (unexpected)
--- FAIL: types.test_slice_wrong_type (unexpected)
--- FAIL: test.transaction_rollback (unexpected)
--- FAIL: types.test_i8_params (unexpected)
--- FAIL: types.test_i64_params (unexpected)
--- FAIL: types.test_f64_params (unexpected)
--- FAIL: types.test_name_params (unexpected)
--- FAIL: types.test_oid_params (unexpected)
--- FAIL: types.test_borrowed_bytea (unexpected)
--- FAIL: types.test_bytea_params (unexpected)
--- FAIL: test.query_prepared (unexpected)
--- FAIL: types.system_time (unexpected)
--- FAIL: types.test_borrowed_text (unexpected)
--- FAIL: types.test_lsn_params (unexpected)
--- FAIL: test.transaction_drop_immediate_rollback (unexpected)
--- FAIL: test.query_unprepared (unexpected)
--- FAIL: types.test_f64_nan_param (unexpected)
--- FAIL: types.test_f32_params (unexpected)
--- FAIL: types.test_text_params (unexpected)
--- FAIL: test.copy_out (unexpected)
--- FAIL: test.explicit_close (unexpected)
--- FAIL: types.test_f32_nan_param (unexpected)
--- FAIL: types.test_varchar_params (unexpected)
--- FAIL: test.transaction_drop (unexpected)
For a full summary look at the rust-postgres artifacts 
An updated blocklist (rustPostgresBlockList) is available in the artifacts' rust-postgres log
test artifacts and logs in: /artifacts/rust-postgres/run_1
```
<p>Parameters: <code>ROACHTEST_arch=amd64</code>
, <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=16</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-12121579/rust-postgres/1696930436560/1696930655358)

</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*rust-postgres.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.rust-postgres [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/12148749?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/12148749?buildTab=artifacts#/rust-postgres) on master @ [619aa6edad0269f75d12cb7d4b88b52f09825c7b](https://github.com/cockroachdb/cockroach/commits/619aa6edad0269f75d12cb7d4b88b52f09825c7b):


```
(orm_helpers.go:212).summarizeFailed: 
Tests run on Cockroach v23.2.0-alpha.4-dev-619aa6edad0269f75d12cb7d4b88b52f09825c7b
Tests run against rust-postgres 
143 Total Tests Run
48 tests passed
95 tests failed
0 tests skipped
1 test ignored
0 tests passed unexpectedly
28 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: types.system_time (unexpected)
--- FAIL: test.query_unprepared (unexpected)
--- FAIL: runtime.cancel_query (unexpected)
--- FAIL: types.test_bpchar_params (unexpected)
--- FAIL: test.transaction_rollback (unexpected)
--- FAIL: test.explicit_close (unexpected)
--- FAIL: types.test_borrowed_text (unexpected)
--- FAIL: types.test_lsn_params (unexpected)
--- FAIL: test.transaction_drop (unexpected)
--- FAIL: test.cancel_query (unexpected)
--- FAIL: types.test_text_params (unexpected)
--- FAIL: types.test_i64_params (unexpected)
--- FAIL: types.test_slice_wrong_type (unexpected)
--- FAIL: types.test_bytea_params (unexpected)
--- FAIL: types.test_i8_params (unexpected)
--- FAIL: types.test_f32_params (unexpected)
--- FAIL: types.test_borrowed_bytea (unexpected)
--- FAIL: types.test_varchar_params (unexpected)
--- FAIL: types.inet (unexpected)
--- FAIL: test.transaction_drop_immediate_rollback (unexpected)
--- FAIL: types.test_name_params (unexpected)
--- FAIL: types.test_bool_params (unexpected)
--- FAIL: types.test_f64_nan_param (unexpected)
--- FAIL: types.test_f64_params (unexpected)
--- FAIL: test.query_prepared (unexpected)
--- FAIL: types.test_f32_nan_param (unexpected)
--- FAIL: types.test_oid_params (unexpected)
--- FAIL: test.copy_out (unexpected)
For a full summary look at the rust-postgres artifacts 
An updated blocklist (rustPostgresBlockList) is available in the artifacts' rust-postgres log
test artifacts and logs in: /artifacts/rust-postgres/run_1
```
<p>Parameters: <code>ROACHTEST_arch=amd64</code>
, <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=16</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-12148749/rust-postgres/1697120122093/1697120324836)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #112171 roachtest: rust-postgres failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-23.2 release-blocker]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*rust-postgres.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

--- a/pkg/cmd/roachtest/tests/rust_postgres.go
+++ b/pkg/cmd/roachtest/tests/rust_postgres.go
@@ -33,7 +33,10 @@ func registerRustPostgres(r registry.Registry) {
 		t.Status("setting up cockroach")
 		c.Put(ctx, t.Cockroach(), "./cockroach", c.All())
 
-		c.Start(ctx, t.L(), option.DefaultStartOptsInMemory(), install.MakeClusterSettings(), c.All())
+		// Most other ORM tests use an in-memory cluster. However, for this test, we
+		// need to restart the cluster with a different port, so we need disk
+		// storage.
+		c.Start(ctx, t.L(), option.DefaultStartOpts(), install.MakeClusterSettings(), c.All())
 		db := c.Conn(ctx, t.L(), 1)
 		_, err := db.Exec("create user postgres with createdb createlogin createrole cancelquery")
 		if err != nil {

